### DISCUSSION TITLE
Mouse disapearing when pressing on Grapgic Tablet

### DISCUSSION THREAD

**Discussion by Sil3ntCat** [ppy/osu#19722] [Q&A] [answered]
I already read some discussions about that problem but that didn't seem to help me.
Currently I'm using the UGEE S640 with the external driver. In other discussions was Windows Ink mentioned, I know if I turn it off the Game does not do that, but if its on mouse mode it is behaving litterally like a mouse and that is pretty akward with a graphic tablet.
Also disabling "Touch" in the Osu settings lets my Cursor just stuck on the place I press.
Im wondering if there is any other way like changing something in the code to let the cursor not fade or disappear?
If yes I would be glad to get a little tutorial through the settings to change that because I'm not really in the coding stuff :/

And I'm also not quite sure how to install the ingame tablet support, do I need to uninstall the external driver to do so ?

Game Version:
osu! 2022.810.2-lazer

Logs:
[database.log](https://github.com/ppy/osu/files/9315854/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9315856/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9315857/network.log)
[performance.log](https://github.com/ppy/osu/files/9315858/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9315859/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9315860/updater.log)

**Comment by peppy** [comment, marked as answer]
There's a setting to always show the cursor which you can use for now.

![osu! 2022-08-12 at 11 32 48](https://user-images.githubusercontent.com/191335/184345790-cabfe6ae-7139-4411-abd0-8ba356bce60e.png)

**Reply by Sil3ntCat** [reply]
Thanks alot, I searched in the settings before and there was no option like that. I'm glad i asked :D
